README.adoc

```diff
@@ -147,8 +147,7 @@ twoSort (2,1) = (1,2)
 The in-Haskell simulator is much slower than Verilator, but can be
 more convenient for small designs.  However, it currently has fewer
 features (no
-http://blarney-lang.github.io/blarney/Blarney-Core-Module.html#t:RegFile[RegFile]
-or <<Modular compilation>>).
+http://blarney-lang.github.io/blarney/Blarney-Core-Module.html#t:RegFile[RegFile]).
```

Comment:
I would delete this whole "however".  No need to scare off users just because of RegFile.  We could add the word "experimental" somewhere if we're not sure it's ready.

Reply:
ok I do that